frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -348,7 +348,7 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
           {selectedInstallMode === InstallModeType.InstallModeTypeAllNamespaces
             ? ' global '
             : ' single namespace '}
-          installation mode.
+          installation mode. Select a different installation namespace that supports this mode.
```

Comment:
I think it would be more helpful to check for the specific case in the reported bug: a user trying to install a single namespace operator into `globalNS`. That's probably the only common use case where this would happen. Then we can give a  clear message like:

> The `openshift-operators` namespace is reserved for global operators that watch all namespaces. To install an operator in a single namespace, select a different namespace where the operand should run.

(If anyone has suggestion to phrase that better, I'm all ears :)

I'd also suggest we never preselect `globalNS` in the dropdown for single-namespace installation. I have a feeling that @smarterclayton was looking at the status of a different operator in `openshift-operators`, then went to OperatorHub where the active namespace was preselected in the form. We shouldn't default to a value that we know won't work. (I've confirmed this is the behavior in 4.6.)

cc @itsptk @tlwu2013

Reply:
@tlwu2013 Can more than one namespace have a global install mode? If so, maybe we stick with the original message change and also include Sam's two suggestions:
1.  Never pre-select `openshift-operators` by default for single-namespace install (instead have a blank selection that the user is forced to select. If blank values aren't possible, maybe default to some other (?) namespace.
2. If the user selects `openshift-operators` for single-namespace install provide the special case message Sam specified.